frontend/src/components/dashboard/create/govsg/GovsgMessages.tsx

```diff
@@ -237,7 +254,18 @@ export const GovsgMessages = ({ campaignId }: GovsgMessagesProps) => {
         <TitleBar
           title={govsgMessageCount + ' Messages'}
           overrideStyles={overrideStylesTitleBar}
-        />
+        >
+          <TextInput
+            value={search}
+            type="text"
+            placeholder="Search for a message"
```

Comment:
Should we make this message more descriptive? Perhaps to inform the user that they can search by recipient name or phone number?

Reply:
This is aligned with the Figma though...

Sure, I can change the description.

Reply:
Oh... hmm maybe can just check with Yi Xin?